### ISSUE TITLE
PHP 5.6 Compatibility

### ISSUE THREAD

**Issue by PanosSynetos** [sabrina-zeidan/speedguard#7]
Hey @sabrina-zeidan - Awesome work!

I wanted to try your plugin, on a PHP 5.6 installation and I get a fatal error. 

```
Fatal error: Default value for parameters with a class type hint can only be NULL in 
/wp-content/plugins/speedguard-master/admin/includes/class.tests.php on line 18
```

This line is the following

```php
 public function prepare_items(string $client_id = ''){
```

As of PHP 7.0, you can typehint `bool`, `float`, `int`, and `string` so, the plugin should state that it's PHP7.0+ compatible.

Thanks
Panos

**Comment by sabrina-zeidan** [comment]
Fixed in 1.8.3 - PHP 5.6 compatible

**Comment by sabrina-zeidan** [comment]
Thank you, Panos!
